org.aposin.licensescout.core/src/main/java/org/aposin/licensescout/finder/JavaJarFinder.java

```diff
@@ -386,14 +393,23 @@ private boolean isPomFile(final String name) {
         return name.endsWith("pom.xml");
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    // TODO: use this value; output it in reports
```

Comment:
Please double check this. If it is a proper TODO open a ticket for it and add the number here. 
Otherwise just delete it.